### PR TITLE
Add Tinido

### PR DESCRIPTION
**Pull request by quangctkm9207**
Add Tinido to Task Management Tool section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,4 +112,4 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 
 ### Miscellaneous
 
-- [ShareX](https://getsharex.com/) – Screen capture, file sharing and productivity tool.
\ No newline at end of file
+- [ShareX](https://getsharex.com/) – Screen capture, file sharing and productivity tool.
```
Comment (jyguyomarch) [inline README.md:115]: Why this change? Just a different EOF? Maybe simply revert it

File: README.md [hunk 2]
```diff
@@ -78,6 +78,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [Taskade](https://taskade.com) – Simple and collaborative task lists for teams.
 - [Taskwarrior](http://taskwarrior.org/) – An open source command line task manager. Flexible, fast, efficient, and unobtrusive.
 - [Timing](https://timingapp.com/) – Automatic time and productivity tracking for Mac. Helps you stay on track with your work and ensures no billable hours get lost if you are billing hourly.
+- [Tinido](https://tinido.com/) - A beautiful task manager app showing your activity as GitHub contribution calendar.
```
Comment (jyguyomarch) [inline README.md:81]: Would suggest to rework the text. Something like "A beautiful and simple task management app that can also give you insights by showing your completed tasks in a contribution graph similar to the one you find on your GitHub profile."

Comment (jyguyomarch) [inline README.md:81]: Use a em dash instead of a normal one

**Review by jyguyomarch — CHANGES_REQUESTED** [review]
Small changes proposed

File: README.md (hunk 2 above)
Comment (quangctkm9207) [reply README.md:81]: Thanks for suggestion

Comment (quangctkm9207) [reply README.md:81]: Got it.

File: README.md (hunk 1 above)
Comment (quangctkm9207) [reply README.md:115]: I edited the README file directly via GitHub page (on browser).

Comment (quangctkm9207) [reply README.md:115]: Sorry for this mess.

File: README.md (hunk 2 above)
Comment (jyguyomarch) [reply README.md]: 👍

**Review by jyguyomarch — APPROVED** [review]
👍

**Pull request merged**